vcstool/outputs.py

```diff
@@ -0,0 +1,18 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class CompareOutput:
+    """Simple dataclass which contains the output of the CompareCommand."""
+
+    local_branch: str
+    remote_branch: str
+    tag: str
```

Comment:
I know `pickles` has this as output but I have found that this is rarely (if even ever) used. The real issues are:
-  There could be multiple tags at a given commit and do we assume to take the latest?
- `vcstool` has an entry in the _repos_ file for version, which can be a tag, and makes this column redundant/confusing

Reply:
Makes sense, you think I should just delete it?

Reply:
Alternatively I can keep it hidden by default, and add a flag to display it if desired